features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -339,7 +359,7 @@ class ApiConsole extends React.Component {
                             </Paper>
                         </Grid>
                     )}
-                    {subscriptions && (
+                    {user != null && subscriptions && (
```

Comment:
Here also, can't we just check `user && subscriptions`..

Reply:
Fixed in 413eb4748a5e0a6ad71d4e9a19db6da8c62c6832